src/views/Clients/ViewClient/index.jsx

```diff
@@ -136,20 +151,48 @@ export default class ViewClient extends Component {
     this.setState({ accessToken: null });
   };
 
+  handleClearAccessTokenCard = () => {
+    this.setState({ accessToken: null, accessTokenCardOpen: false });
+  };
+
   render() {
-    const { error, loading, accessToken } = this.state;
-    const { isNewClient, data } = this.props;
+    const { error, loading, accessToken, accessTokenCardOpen } = this.state;
+    const { isNewClient, data, classes } = this.props;
 
     return (
       <Dashboard title={isNewClient ? 'Create Client' : 'Client'}>
         <Fragment>
           {error && <ErrorPanel error={error} />}
           {accessToken && (
-            <ErrorPanel
-              warning
-              onClose={this.handleAccessTokenWarningClose}
-              error={`The access token for this clientId is ${accessToken}`}
-            />
+            <Fragment>
+              <ErrorPanel
+                warning
+                onClose={this.handleAccessTokenWarningClose}
+                error={`The access token for this clientId is ${accessToken}`}
+              />
+              <Collapse in={accessTokenCardOpen}>
+                <Card>
+                  <CardHeader
+                    className={classes.cardHeader}
+                    action={
+                      <IconButton onClick={this.handleClearAccessTokenCard}>
+                        <ClearIcon />
+                      </IconButton>
+                    }
+                    title={`The access token for this clientId is ${accessToken}`}
+                  />
+                  <CardActions>
+                    <CopyToClipboard
+                      onCopy={this.handleCopyToClipboard}
+                      text={accessToken}>
+                      <Button size="small">
+                        Copy access token to clipboard
+                      </Button>
+                    </CopyToClipboard>
+                  </CardActions>
+                </Card>
+              </Collapse>
+            </Fragment>
```

Comment:
Let's try to stay consistent with what we have in http://localhost:5080/tasks/AZeK_4LNTT2UUHbvBVcjnQ. Something like:

![screen shot 2018-10-17 at 8 46 58 am](https://user-images.githubusercontent.com/3766511/47087149-4b8e4400-d1e9-11e8-826c-56c71cdc9654.png)